pkg/managed/service.go

```diff
@@ -255,15 +255,15 @@ func (m *Service) requiresStackUpdate(nodeGroupName string) (bool, error) {
 		return false, err
 	}
 
-	ver, found, err := manager.GetEksctlVersion(ngStack.Tags)
+	ver, found, err := manager.GetEksctlVersionFromTags(ngStack.Tags)
 	if err != nil {
 		return false, err
 	}
 	if !found {
 		return true, nil
 	}
 
-	curVer, err := semver.ParseTolerant(version.GetVersion())
+	curVer, err := version.ParseEksctlVersion(version.GetVersion())
```

Comment:
`ParseEksctlVersion` is not defined. Did you mean `manager.GetEksctlVersionFromTags`? 

```suggestion
	curVer, err := manager.GetEksctlVersionFromTags(version.GetVersion())
```